### ISSUE TITLE
Update GitHub actions instructions

### ISSUE THREAD

**Issue by domoritz** [intuit/auto#2246]
The instructions at https://intuit.github.io/auto/docs/build-platforms/github-actions still mention `actions/setup-node@v1` even though version 3 is out already. The new version also supports caching so we probably don't need a separate step. The same goes for `actions/checkout@v2` which has a version 3 already. It also has `fetch-depth: 0`, which may be able to replace the extra git step in the instructions.

**Comment by laughedelic** [comment]
There's also https://github.com/auto-it/setup-auto

**Comment by domoritz** [comment]
Nice. I guess that tool lets you use auto without making it a dependency? However, I don't see instructions for setting up NPM there.

**Comment by laughedelic** [comment]
Yes, you don't need Auto as a dependency. That action installs Auto standalone binary, so it's ready to be called from shell-steps, e.g. `auto shipit`. It's not npm-projects specific, it should work for any project, including npm.

**Comment by domoritz** [comment]
Yes, but if you want NPM, you need to setup the tokens as well and the readme at https://github.com/auto-it/setup-auto/blob/main/README.md doesn't describe how to do that. It would be nice to add some instructions/examples for releasing to NPM.
